backend/tournesol/views/user.py

```diff
@@ -23,3 +30,29 @@ def delete(self, request):
         logout(request)
         logger.info("User '%s' with email '%s' has been deleted.", user.username, user.email)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+
+
+class UserDataDumpView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename="export.csv"'
+        writer = csv.DictWriter(response, fieldnames=['video_a', 'video_b', 'criteria', 'weight', 'score'])
+        writer.writeheader()
```

Comment:
We need also to test if headers are correctly written, and readable for a human

Reply:
As we now use the csv.DictReader in the test, the test includes parsing the column names, then makes sure that "scores" is one of the columns